Still get:
```
error: builder for '/nix/store/ra4dcr4xh4gk30mhsa91y4ajqw7smhdi-python3.13-netbox-topology-views-4.1.0.drv' failed with exit code 1;
       last 25 log lines:
       > Running phase: pythonRemoveBinBytecodePhase
       > Running phase: pythonImportsCheckPhase
       > Executing pythonImportsCheckPhase
       > Check whether the following modules can be imported: netbox_topology_views
       > Traceback (most recent call last):
       >   File "<string>", line 1, in <module>
       >     import sys; import importlib; list(map(lambda mod: importlib.import_module(mod), sys.argv[1:]))
       >                                   ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
       >   File "<string>", line 1, in <lambda>
       >     import sys; import importlib; list(map(lambda mod: importlib.import_module(mod), sys.argv[1:]))
       >                                                        ~~~~~~~~~~~~~~~~~~~~~~~^^^^^
       >   File "/nix/store/acxvfb890g9jawlxsj7zs8marxxvzrrs-python3-3.13.1/lib/python3.13/importlib/__init__.py", line 88, in import_module
       >     return _bootstrap._gcd_import(name[level:], package, level)
       >            ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
       >   File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
       >   File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
       >   File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
       >   File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
       >   File "<frozen importlib._bootstrap_external>", line 1026, in exec_module
       >   File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
       >   File "/nix/store/riff192xqj2zw4s4gmakzg112hs6sih3-python3.13-netbox-topology-views-4.1.0/lib/python3.13/site-packages/netbox_topology_views/__init__.py", line 1, in <module>
       >     from netbox.plugins import PluginConfig
       >   File "/nix/store/ilnv7bqc4whj0smxl872w4wy3w8lzvrh-netbox-4.1.7/opt/netbox/netbox/netbox/plugins/__init__.py", line 4, in <module>
       >     from django.apps import AppConfig
       > ModuleNotFoundError: No module named 'django'
       For full logs, run 'nix log /nix/store/ra4dcr4xh4gk30mhsa91y4ajqw7smhdi-python3.13-netbox-topology-views-4.1.0.drv'.
error: 1 dependencies of derivation '/nix/store/mn76v02x38a6h320g8fav7bvzhmn33zb-review-shell.drv' failed to build
--------- Report for 'x86_64-linux' ---------
2 packages failed to build:
python313Packages.netbox-topology-views python313Packages.netbox-topology-views.dist

```

--- a/pkgs/development/python-modules/netbox-topology-views/default.nix
+++ b/pkgs/development/python-modules/netbox-topology-views/default.nix
@@ -1,0 +1,41 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  netbox,
+  pythonAtLeast,
+}:
+buildPythonPackage rec {
+  pname = "netbox-topology-views";
+  version = "4.1.0";
+  pyproject = true;
+
+  disabled = pythonAtLeast "3.13";
+
+  src = fetchFromGitHub {
+    owner = "netbox-community";
+    repo = "netbox-topology-views";
+    rev = "v${version}";
+    hash = "sha256-4ehIF6r4fCgBAaHImzofdQIywtD7ITQFP6DkHXHKMro=";
+  };
+
+  build-system = [ setuptools ];
+
+  nativeCheckInputs = [ netbox ];
+
+  preFixup = ''
+    export PYTHONPATH=${netbox}/opt/netbox/netbox:$PYTHONPATH
+  '';
+
+  pythonImportsCheck = [ "netbox_topology_views" ];
+
+  meta = {
+    description = "Netbox plugin for generate topology views/maps from your devices";
+    homepage = "https://github.com/netbox-community/netbox-topology-views";
+    changelog = "https://github.com/netbox-community/netbox-topology-views/releases/tag/${src.rev}";
+    license = lib.licenses.asl20;
+    platforms = lib.platforms.linux;
+    maintainers = with lib.maintainers; [ felbinger ];
+  };
+}

--- a/pkgs/top-level/python-packages.nix
+++ b/pkgs/top-level/python-packages.nix
@@ -9168,6 +9168,8 @@ self: super: with self; {
 
   netbox-reorder-rack = callPackage ../development/python-modules/netbox-reorder-rack { };
 
+  netbox-topology-views = callPackage ../development/python-modules/netbox-topology-views { };
+
   netcdf4 = callPackage ../development/python-modules/netcdf4 { };
 
   netdata = callPackage ../development/python-modules/netdata { };

